### PR TITLE
first commit added file

### PR DESCRIPTION
**Pull request by shamusharkin**
closes issue #62 which i created at the start.
So, we created an issue (which does not have to be an issue, but can be a piece of work).
Then we create a branch in github for the piece of work, when we do a git clone (or pull if we already have the project locally) the branch will be available locally.
to merge the branch back into master
first we create a pull request......and then we will see how the rest goes......

### CODE REVIEW

File: shamusharkinderry.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Derry ireland
```
Comment (Shanker1234) [inline shamusharkinderry.md]: Which country is it?

Comment (Shanker1234) [inline shamusharkinderry.md]: :100:

**Comment by shamusharkin** [comment]
Happy for the feedback

**Review by Shanker1234 — APPROVED** [review]
This looks great.

**Comment by shamusharkin** [comment]
Code review complete.  merging code into master

**Pull request merged**